examples/classic/distributed/TwoPhaseCommit/two_phase_commit.qnt

```diff
@@ -0,0 +1,162 @@
+// -*- mode: Bluespec; -*-
+
+/* This specification stems from the specification of the two phase protocol
+   by Leslie Lamport:
+
+   https://github.com/tlaplus/Examples/blob/master/specifications/transaction_commit/TwoPhase.tla */
+
+module two_phase_commit {
+  type ResourceManagerState = Working | Prepared | Committed | Aborted
+  var resourceManagerStates: str -> ResourceManagerState
+
+  type TransactionManagerState = Init | Done
+  var transactionManagerState: TransactionManagerState
+
+  /// The set of resource managers from which the transaction manager has
+  /// received RMPrepared messages
+  var preparedResourceManagers: Set[str]
```

Comment:
Makes sense. I think I was subconsciously trying to make the point that this partition of state space should be clear in the spec, while currently it is not. The lack of that information in the name makes this a stronger example for the RFC I'm working on :laughing:.

I'll improve the names in this spec for merging, but I think I can still use this version of names in the RFC examples.

Reply:
I might have exaggerated here, but I tried adding `RM::` and `TM::` prefixes to variables and actions, and I actually like how it turned out. Can you have a look and see how you feel about it? @thpani @shonfeder 

Also, I'm considering renaming the `*state` types and variables to `*status`, because the word state is a bit overloaded in the sense of a state machine's state. WDYT?